protobuf.bzl

```diff
@@ -193,6 +193,30 @@ def cc_proto_library(
       **kargs)
 
 
+def internal_gen_well_known_protos_java(srcs):
+  """Bazel rule to generate the gen_well_known_protos_java genrule
+
+  Args:
+    srcs: the well known protos
+  """
+  root = Label("%s//protobuf_java" % (REPOSITORY_NAME)).workspace_root
```

Comment:
this rule seems to be only used in the java well known types generation. can this be part of the genrule directly in the BUILD?